doc/source/release/1.11.0-notes.rst

```diff
@@ -105,25 +267,837 @@ Deprecated features
 ******************************
 Backwards incompatible changes
 ******************************
+- The default for the ``legacy`` keyword of `scipy.special.comb` has changed
+  from ``True`` to ``False``, as announced since its introduction.
+
+********************
+Expired Deprecations
+********************
+There is an ongoing effort to follow through on long-standing deprecations.
+The following previously deprecated features are affected:
+
+- The ``n`` keyword has been removed from `scipy.stats.moment`.
+- The ``alpha`` keyword has been removed from `scipy.stats.interval`.
+- The misspelt ``gilbrat`` distribution has been removed (use
+  `scipy.stats.gibrat`).
+- The deprecated spelling of the ``kulsinski`` distance metric has been
+  removed (use `scipy.spatial.distance.kulczynski1`).
+- The ``vertices`` keyword of `scipy.spatial.Delauney.qhull` has been removed
+  (use simplices).
+- The ``residual`` property of `scipy.sparse.csgraph.maximum_flow` has been
+  removed (use ``flow``).
+- The ``extradoc`` keyword of `scipy.stats.rv_continuous`,
+  `scipy.stats.rv_discrete` and `scipy.stats.rv_sample` has been removed.
+- The ``sym_pos`` keyword of `scipy.linalg.solve` has been removed.
+- The `scipy.optimize.minimize` function now raises an error for ``x0`` with
+  ``x0.ndim > 1``.
+- In `scipy.stats.mode`, the default value of ``keepdims`` is now ``False``,
+  and support for non-numeric input has been removed.
+- The function `scipy.signal.lsim` does not support non-uniform time steps
+  anymore.
+
 
 *************
 Other changes
 *************
+- Rewrote the source build docs and restructured the contributor guide.
+- Improved support for cross-compiling with meson build system.
+- MyST-NB notebook infrastructure has been added to our documentation.
 
 
 
 *******
 Authors
 *******
 
+* h-vetinari (69)
+* Oriol Abril-Pla (1) +
+* Anton Akhmerov (13)
+* Andrey Akinshin (1) +
+* alice (1) +
+* Oren Amsalem (1)
+* Ross Barnowski (11)
+* Christoph Baumgarten (2)
+* Dawson Beatty (1) +
+* Doron Behar (1) +
+* Peter Bell (1)
+* John Belmonte (1) +
+* boeleman (1) +
+* Jack Borchanian (1) +
+* Matt Borland (3) +
+* Jake Bowhay (40)
+* Sienna Brent (1) +
+* Matthew Brett (1)
+* Evgeni Burovski (38)
+* Matthias Bussonnier (2)
+* Maria Cann (1) +
+* Alfredo Carella (1) +
+* CJ Carey (17)
+* Hood Chatham (2)
+* Anirudh Dagar (3)
+* Alberto Defendi (1) +
+* Pol del Aguila (1) +
+* Hans Dembinski (1)
+* Dennis (1) +
+* Vinayak Dev (1) +
+* Thomas Duvernay (1)
+* DWesl (4)
+* Stefan Endres (66)
+* Evandro (1) +
+* Tom Eversdijk (2) +
+* Isuru Fernando (1)
+* Franz Forstmayr (3)
+* Joseph Fox-Rabinovitz (1)
+* Franz (1) +
```

Comment:
https://github.com/scipy/scipy/commit/5acb9564989ea45706e1d175dfd7a49479d359f9 has a wrong (or incomplete) author name (not visible in GH UI, so here directly from git):
```
commit 5acb9564989ea45706e1d175dfd7a49479d359f9
Author: Franz <forstmayr.franz@gmail.com>
Date:   Tue May 16 00:32:58 2023 +0200

    DEP: `lsim2` deprecated in favor of `lsim` (#15929)

    likewise for step2 & impulse2
```


it should be:
```suggestion
* Franz Forstmayr (4)
* Joseph Fox-Rabinovitz (1)
```
Guess we need to do this through mailmap?